src/main/resources/db/migration/0300/V348__FundingEntities.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE accelerator.funding_entities
+(
+    id   BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL,
```

Comment:
Do funding entity names need to be unique?

Reply:
Good question, have a question out to product on this one

Reply:
Jennifer just confirmed they need to be unique. Updating this